Makefile

```diff
@@ -1,9 +1,23 @@
+detect_long_lines_in_diff:
```

Comment:
Any way to move this to a external bundle? This would reduce code when using in other bundles.